xbmc/Application.cpp

```diff
@@ -238,35 +238,36 @@ using KODI::MESSAGING::HELPERS::DialogResponse;
 
 //extern IDirectSoundRenderer* m_pAudioDecoder;
 CApplication::CApplication(void)
-:
+    :
 #ifdef HAS_DVD_DRIVE
-  m_Autorun(new CAutorun()),
+    m_Autorun(new CAutorun())
+    ,
 #endif
-  m_iScreenSaveLock(0)
-  , m_confirmSkinChange(true)
```

Comment:
those should be removed and init done c++11 style in header

Reply:
This is only about formatting